corehq/apps/locations/views.py

```diff
@@ -819,7 +819,7 @@ def products_form(self):
     def users_form(self):
         if not (self.can_edit_commcare_users or self.can_access_all_locations):
             return None
-        if toggles.LOCATION_HAS_USERS.enabled(self.domain) and not self.location.location_type.has_users:
+        if not self.location.location_type.has_users:
```

Comment:
For the conditional checks that no longer checks a FF, what happens if `USH_RESTORE_FILE_LOCATION_CASE_SYNC_RESTRICTION` is turned off? Will these locations start showing?

Reply:
Hm so it seems like when `USH_RESTORE_FILE_LOCATION_CASE_SYNC_RESTRICTION ` is turned off, some of behavior associated with the previous `LOCATION_HAS_USERS` FF is preserved while others are not. I'm thinking it should be all or none (meaning to continue checking for the `USH_RESTORE_FILE_LOCATION_CASE_SYNC_RESTRICTION` FF in the first two places you mentioned).

Because right now with the `USH_RESTORE_FILE_LOCATION_CASE_SYNC_RESTRICTION` FF off, the "Has User" UI is hidden meaning that there won't be a way to reenable a location as having user? So it won't be possible for users to be added to a location

Reply:
Ya my main concern was leaving the domain in a weird/buggy state when the FF is off. But I also agree with your other points. I don't feel too strongly about it so either way is fine